packages/peregrine/lib/Router/magentoRouteHandler.js

```diff
@@ -112,8 +112,12 @@ export default class MagentoRouteHandler extends Component {
 
             // associate the matching RootComponent with this location
             this.setRouteComponent(pathname, RootComponent, { id });
-        } catch ({ message }) {
-            const symbol = message === '404' ? NotFound : InternalError;
+        } catch (e) {
+            if (process.env.NODE_ENV === 'development') {
+                console.error(e);
+            }
+
+            const symbol = e.message === '404' ? NotFound : InternalError;
```

Comment:
This is a good change. Thank you. We should be doing this everywhere.

Reply:
I think this is your code haha